javascript/node/selenium-webdriver/chromium.js

```diff
@@ -738,20 +738,28 @@ class Driver extends webdriver.WebDriver {
   * Doesn't return anything, sets the listener and goes off.
   */
   async register(username, password) {
-    this.sendDevToolsCommand('Fetch.enable', {
+    await this.sendDevToolsCommand('Fetch.enable', {
       handleAuthRequests: true
     })
 
-    this._wsConnection.on('Fetch.authRequired', (params) => {
-      this.sendDevToolsCommand('Fetch.continueWithAuth', {
-        requestId: params['requestId'],
-        authChallengeResponse: {
-          response: 'ProvideCredentials',
-          username: username,
-          password: password
-        }
-      });
+    this._wsConnection.on('message', (message) => {
+      console.log(message);
```

Comment:
is this required @raju249 , or just placed for debugging purpose ?

Reply:
Hmm, no. But I think I have removed it before, maybe it was old diff that you saw?

Reply:
may be Yes please ignore!